internal/config/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	URLFilters           []*regexp.Regexp  `env:"URL_RULES" envSeparator:","`
 	DisallowedURLFilters []*regexp.Regexp  `env:"DISALLOWED_URL_RULES" envSeparator:","`
 	SkipValidation       bool              `env:"SKIP_VALIDATION" envDefault:"false"`
+	Async				 bool              `env:"ASYNC" envDefault:"true"`
```

Comment:
This line is not using the same spacing characters as the rest of the lines above it.

I'd suggest running `go fmt` which will sort it out

Reply:
ok, will do